www/src/pages/slider.mdx

```diff
@@ -0,0 +1,274 @@
+import Helmet from "react-helmet"
+
+<Helmet
+  title="Reach UI - Slider"
+  meta={[
+    {
+      name: "description",
+      content: "Accessible Slider for React.js",
+    },
+  ]}
+/>
+
+# Slider
+
+- Source: https://github.com/reach/reach-ui/tree/master/packages/slider
+- WAI ARIA: https://www.w3.org/TR/wai-aria-practices-1.1/#slider
+
+A UI input component where the user selects a value from within a given range. A Slider has a handle that can be moved along a track to change its value. When the user's mouse or focus is on the Slider's handle, the value can be incremented with keyboard controls.
+
+Sliders are composed by an outer `Slider` component with a `SliderTrack` child component. `SliderTrack` can accept `SliderTrackHighlight`, `SliderMarker` and `SliderHandle` components as children.
+
+_Pointer Events_: Slider uses pointer events for unified event handling across different pointer types (mouse, touch, stylus, etc.). As such, to override default behavior you will need to use pointer events rather than mouse or touch events. For example, if you wish to call `event.preventDefault` to override the slider handle's activation on a mouse click event, you'll want to reach for `onPointerDown` instead of `onMouseDown`. If you need to support older browsers without native pointer events, we recommend [`pepjs`](https://github.com/jquery/PEP) as a solid polyfill.
```

Comment:
We should probably say "older browsers or Safari"

Reply:
Added some nuance here about Safari. Pointer events are now fully supported on desktop since 13.1, and mostly supported on iOS with some minor issues that shouldn't impact our usage.